org.eclipse.egit.github.core/src/org/eclipse/egit/github/core/Notification.java

```diff
@@ -0,0 +1,110 @@
+/*******************************************************************************
+ *  Copyright (c) 2011 GitHub Inc.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  Contributors:
+ *    Kevin Sawicki (GitHub Inc.) - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.egit.github.core;
+
+import java.io.Serializable;
+import java.util.Date;
+
+public class Notification implements Serializable {
+
+	/**
+	 * serialVersionUID
+	 */
+	private static final long serialVersionUID = 989467705847081308L;
+
+	private String id;
```

Comment:
Probably not, [API](https://developer.github.com/v3/activity/notifications/#list-your-notifications) displays it in double quotes. Or is it somehow converted on egit side?